terragrunt/aws/cloudfront/route53.tf

```diff
@@ -0,0 +1,43 @@
+resource "aws_route53_record" "url_shortener_A" {
+  zone_id = var.hosted_zone_id
+  name    = var.domain
+  type    = "A"
+
+  alias {
+    name                   = aws_cloudfront_distribution.url_shortener_api.domain_name
+    zone_id                = aws_cloudfront_distribution.url_shortener_api.hosted_zone_id
+    evaluate_target_health = false
+  }
+}
+
+resource "aws_route53_health_check" "url_shortener_A" {
+  fqdn              = aws_route53_record.url_shortener_A.fqdn
+  port              = 443
+  type              = "HTTPS"
+  resource_path     = "/healthcheck"
+  failure_threshold = "5"
+  request_interval  = "30"
+  regions           = ["us-east-1", "us-west-1", "us-west-2"]
+
+  tags = {
+    CostCentre = var.billing_code
+    Terraform  = true
+  }
+}
```

Comment:
For now I think we can remove this.  I initially had it setup on the GitHub secret scanning project too but decided to scale back and just use the [Status page](https://github.com/cds-snc/status-statut).

My thinking was this causes a lot of invocations of the function that we don't necessarily want to pay for.   But on the other hand, it would also be fairly cheap in the scheme of things.  What do you think?

Reply:
I have removed it and I agree with you - there is no reason to pay for it especially since I am not sure if we will use that endpoint that much anyways and really the status page will be more useful. We can always add it later to the code if we want to. But I will add a ticket in the queue to add the url shortener on the Status page once it is up and running.